account_tax_balance/wizard/open_tax_balances.py

```diff
@@ -9,7 +9,7 @@ class WizardOpenTaxBalances(models.TransientModel):
     _description = "Wizard Open Tax Balances"
 
     company_id = fields.Many2one(
```

Comment:
Shouldn't this be now a m2m?

Reply:
Yes could be. In this case is not only a fix but also an implementation of a new feature.

Reply:
Well, it's the multi-company implementation of v13 in fact.

Reply:
Done